docs/docs/configuration/additional-mapping-parameters.mdx

```diff
@@ -0,0 +1,58 @@
+---
+sidebar_position: 6
+description: Additional mapping parameters
+---
+
+# Additional mapping parameters
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+A mapping method declaration can have additional parameters.
+Each additional parameter is considered the same as a source member and matched by its case-insensitive name.
+An additional mapping parameter has lower priority than a `MapProperty` mapping,
+but higher than a by-name matched regular member mapping.
+
+<Tabs>
+  <TabItem default label="Declaration" value="declaration">
+    ```csharp
+    [Mapper]
+    public partial class CarMapper
+    {
+      // highlight-start
+      public partial CarDto Map(Car source, string name);
```

Comment:
Maybe this example could benefit from showing the `Car` and `CarDto` definitions

Reply:
Added.